dms_account_avatax_exemption/models/exemption.py

```diff
@@ -0,0 +1,14 @@
+from odoo import fields, models
+
+
+class ResPartnerExemption(models.Model):
+    _name = "res.partner.exemption"
+    _description = "Res Partner Exemption"
+    _inherit = [
+        "res.partner.exemption",
+        "abstract.dms.mixin",
+    ]
```

Comment:
```suggestion
    _name = "res.partner.exemption"
    _description = "Res Partner Exemption"
    _inherit = [
        "res.partner.exemption",
        "abstract.dms.mixin",
    ]
```

why not just simply inherit from `res.partner.exemption`, what is the benefit of inheriting `abstract.dms.mixin` as well

Reply:
I remember I needed a field or a method based on mixin but not remember exactly, so I am using mixin.